websites/N/NationStates/presence.ts

```diff
@@ -0,0 +1,273 @@
+const presence = new Presence({
+		clientId: "1006869424441131109",
+	}),
+	browsingTimestamp: number = Math.floor(Date.now() / 1000),
+	presenceData: PresenceData = {
+		largeImageKey: "ns",
```

Comment:
optional but we prefer imgur links here
```suggestion
		largeImageKey: "https://i.imgur.com/04gnehi.png",
```

Reply:
Done, all Discord assets replaced with imgur links; commit [#d250ef7](https://github.com/PreMiD/Presences/pull/6634/commits/d250ef7d10defdf54a5eaf9334b702207dc7721a)